src/experimental/tools/mediaCapabilitiesProber/probers/mediaDisplayInfos/index.ts

```diff
@@ -48,7 +48,7 @@ export default function probeMatchMedia(
             "Bad arguments for calling matchMedia.");
         }
         const result = match.matches && match.media !== "not all" ? 2 : 0;
-        return result;
+        return [result] as [number];
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=315&issues=AWXy2D-32IDoRcZ5Haqq&open=AWXy2D-32IDoRcZ5Haqq)

[ISSUE_KEY]: AWXy2D-32IDoRcZ5Haqq